src/pl/tomaszosuch/MainCalculatorSalary.java

```diff
@@ -0,0 +1,17 @@
+package pl.tomaszosuch;
+
+import pl.tomaszosuch.calculator.SalaryCalculator;
+import pl.tomaszosuch.calculator.SalaryCalculatorConfig;
+import pl.tomaszosuch.model.Employee;
+
+public class MainCalculatorSalary {
+
+    public static void main(String[] args) {
+        Employee employee = new Employee("Jan", "Kowalski", 14.93, 8);
+        SalaryCalculatorConfig calculatorConfig = new SalaryCalculatorConfig(0.0976, 0.015, 0.0245);
```

Comment:
1. Gdy patrzysz na tę linię, widać że podajesz kilka argumentów podobnego typu. Łatwo się pomylić (np. zmienić kolejność). Spróbuj zastosować wzorzec `builder` do tworzenia obiektu konfiguracyjnego (hasło do wygooglania). Dzięki temu będzie "widać", która liczba dotyczy której wartości konfiguracyjnej.
2. To powinny być `BigDecimal`-e.